easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart

```diff
@@ -134,13 +133,11 @@ class NFTsListTile extends StatelessWidget {
                     ),
                   ),
                 ),
-              )
-            : Padding(
+              ) else Padding(
                 padding: EdgeInsets.symmetric(horizontal: 20.w),
                 child: getPublishedCard(context: context),
               ),
-        (publishedNFT.assetType.toAssetTypeEnum() != AssetType.ThreeD)
-            ? IgnorePointer(
+        if (publishedNFT.assetType.toAssetTypeEnum() != AssetType.ThreeD) IgnorePointer(
```

Comment:
IgnorePointer widget should start from new line.

Reply:
No necessary. Thats the thing that formatter decides.